src/mon/MgrStatMonitor.cc

```diff
@@ -79,7 +79,9 @@ void MgrStatMonitor::update_from_paxos(bool *need_bootstrap)
   version = get_last_committed();
   dout(10) << " " << version << dendl;
   bufferlist bl;
-  get_version(version, bl);
+  int err = get_version(version, bl);
```

Comment:
We can invoke update_from_paxos() without ever having committed a version to the underlying LevelDB before. I presume that's why the make check builder failed (although the archives have been deleted).